apps/projects/app/components/Content/Issues.js

```diff
@@ -210,9 +208,26 @@ class Issues extends React.PureComponent {
     this.setState({ showIssueDetail: false, currentIssue: null })
   }
 
-  actionsMenu = () => (
-    <div>
+  disableFilter = (pathToFilter) => {
+    let newFilters = { ...this.state.filters }
+    recursiveDeletePathFromObject(pathToFilter, newFilters)
```

Comment:
This function may seem a little confusing, but really all it's doing is taking a path from an `alias` (in the `ActiveFilters.js` component, and deleting that filter from state (removing the filter).

SO object:
```
{
path: { to: { object: 'value', key: 'value2' }}
}
```
with path: `['path', 'to', 'object']`

becomes:
{
path: { to: { key: 'value2' }}
}
```

thus deleting the filter